app/code/Magento/Inventory/Test/Acceptance/LICENSE.txt

```diff
@@ -0,0 +1,48 @@
+
+Open Software License ("OSL") v. 3.0
```

Comment:
Is it really needed to have README and LICENSE for every test directory? Usually we have jsut one of these files in the root of the module

Reply:
Yes it is. All MFTF tests will be excluded when packages will be created. Separate package will be created only with MFTF tests. And this files are going to represent License and all additional info in that separate package.